services/user.py

```diff
@@ -0,0 +1,51 @@
+from django.contrib.auth import get_user_model
+
+from db.models import User
+
+
+def create_user(
+        username: str,
+        password: str,
+        email: str = "",
+        first_name: str = "",
+        last_name: str = ""

```

Comment:
when some information was not passed it should be None not ""

Reply:
Ok i will fix this but if i use none like default variable, then I must check all attribute for existence like
if email: ...
if first_name:...
etc.
if I use just "" I don't need check it all )

Reply:
okay, I considered this solution again and think that it's also good